src/queries/enum_structvariant_field_missing.ron

```diff
@@ -0,0 +1,66 @@
+SemverQuery(
+    id: "enum_structvariant_field_missing",
+    human_readable_name: "pub enum structvariant's field removed or renamed" ,
+    description: "An enum variant that is a struct has a field that is no longer available under its prior name.",
```

Comment:
These descriptions are really hard to write for me too. Here's a suggestion for you to consider -- this seems a bit more readable to me.
```suggestion
    description: "An enum's struct variant has a field that is no longer available under its prior name.",
```